httpserver/main.cpp

```diff
@@ -79,12 +79,14 @@ int main(int argc, char* argv[]) {
     }
 
     if (options.ip == "127.0.0.1") {
-        LOKI_LOG(error, "Tried to bind loki-storage to localhost, please bind to outward facing address");
+        LOKI_LOG(error, "Tried to bind loki-storage to localhost, please bind "
```

Comment:
Should these be critical? Or what definition of critical are we going with?

Reply:
updated to critical